### ISSUE TITLE
Clean install windows 10 Pro N and JF, no playback of movies possible anymore 

### ISSUE THREAD

**Issue by Japhys** [jellyfin/jellyfin#4866]
Happens with all files. MP4, MKV ...no matter which i try. They all played fine before clean install windows. Seems to be a transcoding problem but I have no idea how to fix it. Everything is straight "out of the box" so you would expect it to work. It's is driving me nuts. I even installed kodi and Serviio to see if that worked and both did. Don't want to use these though :)

**Logs**
`[2020-12-25 20:49:08.874 +01:00] [ERR] [38] Emby.Server.Implementations.HttpServer.HttpListenerHost: Error processing request: "Could not find file 'C:\ProgramData\Jellyfin\Server\transcodes\772bd83514633219c14c9ec22a230664.webm'". URL: "http://192.168.1.124:8096/videos/2078a1fd-619f-b6e8-f8fc-003600aae9ef/stream.webm?`

**FFmpeg Logs**
```
http://192.168.1.124:8096/videos/2078a1fd-619f-b6e8-f8fc-003600aae9ef/stream.webm?DeviceId=TW96aWxsYS81LjAgKFdpbmRvd3MgTlQgMTAuMDsgV2luNjQ7IHg2NDsgcnY6ODQuMCkgR2Vja28vMjAxMDAxMDEgRmlyZWZveC84NC4wfDE2MDg5MjM5NjYyOTM1&MediaSourceId=2078a1fd619fb6e8f8fc003600aae9ef&VideoCodec=vpx&AudioCodec=vorbis&VideoBitrate=139808000&AudioBitrate=192000&PlaySessionId=0754365dc34541d99dea25de077a0b6f&api_key=4ae7f55f415144c8aba5d0770517bdb2&TranscodingMaxAudioChannels=2&RequireAvc=false&Tag=23a5db74ecec11b9e8da487971a418e3&allowVideoStreamCopy=false&allowAudioStreamCopy=false&allowAudioStreamCopy=false

{"Protocol":"File","Id":"2078a1fd619fb6e8f8fc003600aae9ef","Path":"---.mp4","Type":"Default","Container":"mp4","Name":"---(2019)","IsRemote":false,"ETag":"23a5db74ecec11b9e8da487971a418e3","ReadAtNativeFramerate":false,"IgnoreDts":false,"IgnoreIndex":false,"GenPtsInput":false,"SupportsTranscoding":true,"SupportsDirectStream":true,"SupportsDirectPlay":true,"IsInfiniteStream":false,"RequiresOpening":false,"RequiresClosing":false,"RequiresLooping":false,"SupportsProbing":true,"VideoType":"VideoFile","MediaStreams":[],"MediaAttachments":[],"Formats":[],"RequiredHttpHeaders":{}}

C:\Program Files\Jellyfin\Server\ffmpeg.exe -f mp4 -i file:"---.mp4" -sn -codec:v:0 libvpx -force_key_frames "expr:gte(t,n_forced*5)" -pix_fmt yuv420p -speed 16 -quality good -profile:v 0 -slices 8 -crf 10 -qmin 0 -qmax 50 -maxrate:v 139808000 -bufsize:v 279616000 -b:v 139808000 -vsync -1 -map_metadata -1 -map_chapters -1 -threads 3 -codec:a:0 libvorbis -ac 2 -ab 192000  -y "C:\ProgramData\Jellyfin\Server\transcodes\772bd83514633219c14c9ec22a230664.webm"
```

**Stats for Nerds Screenshots**
Weird translation btw!
![2020-12-25_205206](https://user-images.githubusercontent.com/24695937/103141331-2663bd00-46f3-11eb-9290-af73ad02300e.png)

**Server System (please complete the following information):**
 - Windows 10 Pro N (dec 2020), Media pack installed, all drivers up to date (checked)
 - Jellyfin Version: 10.6.0 to 10.6.4
 -  Installed as service (both options) and the other way
 - Intel i5 Intel Iris Graphics 540, 16GB ram

 
**Client System (please complete the following information):**
Happens on all devices
ipad and JF app
Windows Firefox and Chrome
Android Lineageos and JF app

Edit: I though music files worked but they don't. Duration is 0 for all files.

![Screenshot_20201226-175651_Gelli.jpg](https://user-images.githubusercontent.com/24695937/103156034-f5dc5b80-47a4-11eb-9934-6039ea285ddd.jpg)

**Comment by Japhys** [comment]
This is driving me insane! I have been using Emby and Jellyfin forever and now all of a sudden this. I can't see what I did differently, there must be something I am missing but what?? Seems like an ffmpeg problem. But the file is right there in my server folder. Could it have to do with permissions somehow? I am using Network service account now

**Comment by nielsvanvelzen** [comment]
Did you install jellyfin as a service?

**Comment by Japhys** [comment]
I did! But yesterday I tried a regular install and it didn't work either. On my previous windows install I am pretty sure I ran it as a service as well.

**Comment by Japhys** [comment]
@nielsvanvelzen Thanks to you I found the problem. I am happy! 

For future reference:

Conclusion first: The problem is Windows 'N'!

I wanted to give your suggestion a try and disabled the service. After that I started the application the regular way.
I Then got this error:

![2020-12-26_214128](https://user-images.githubusercontent.com/24695937/103159201-bcb4e300-47c6-11eb-97cb-e39a193cf4b3.png)

After Googling a bit I realized this file is missing is the N versions because it doesn't come with the media feature pack. Still,  yesterday I installed the pack (just shooting in the dark) by downloading it from the MS site and that didn't help. This time I installed it in Windows 10 itself (didn't know I could). And there you go, all is playing fine now!

![2020-12-26_215939](https://user-images.githubusercontent.com/24695937/103159202-c0486a00-47c6-11eb-957e-245180ffa0b7.png)

So why did it work with my previous windows installation? Well, I didn't use the N version!
